src/index.js

```diff
@@ -131,6 +131,11 @@ export type RequestInfo = {
    * The result of executing the operation.
    */
   result: ?mixed,
+
+  /**
+   * A value to pass as the context to the graphql() function.
```

Comment:
I think the reference to `graphql` is confusing maybe just `The value passed as the context to this middleware` or something similar. What do you think?

Reply:
Sure, that sounds fine to me too.